libs/openFrameworks/types/ofPoint.h

```diff
@@ -1,9 +1,9 @@
 #pragma once
 
-#include "ofVec3f.h"
+#include "ofMathConstants.h"
 
 /// \file 
 /// ofPoint is a typedef (alias) of ofVec3f
 
 /// \brief Look at ofVec3f for documentation
-typedef ofVec3f ofPoint;
+typedef ofDefaultVec3 ofPoint;
```

Comment:
not so sure about this, i left it like it was on purpose so it's not up to date and people start to use glm instead. Does it help with something in particular? otherwise it shouldn't change and much less in the middle of the release candidate process.

Reply:
I see. Maybe it should have some kind of deprecation warning?
there are lots of parts, both in the core and in the examples, that use ofPoint. Not sure if this helps or not but I feel it is more correct having ofDefaultVec3, although a much better way would be to have ofPoint removed from everywhere and having this just for backwards compatibility. I can do all the changes from ofPoint to glm::vec3. should I do?

Reply:
As far as i see though ofPoint is only used in the core for compatibility reasons in a couple of places in ofParameter but nowhere else? there's a coouple of coments still mentioning ofPoint if you want to change those that's fine but other than that i don't want to change any code except for fixing bugs at this moment. Can you rollback this change to ofVec3?

We can discuss what's the best way going forward after the release but right now we shouldn't be changing anything unless it's a bug or stops something else from working

Reply:
I just reverted it.
Right it is used only on ofParameter and several of comments, although it is still used on several examples.
best

Reply:
Yes examples and addons still have to be updated to use glm

Reply:
I'll go through those and update the PR.
I will make a different PR for the addons.

Reply:
@arturoc Just updated all the mac examples to use glm.